js/components/main.js

```diff
@@ -86,7 +86,7 @@ class Main extends ImmutableComponent {
         location: url || Config.defaultUrl,
         isPrivate: !!options.isPrivate,
         isPartitioned: !!options.isPartitioned
-      })
+      }, options.openInForeground)
```

Comment:
You'll have to either update all the callers of `SHORTCUT_NEW_FRAME` to preserve their old behaviour, or do:
`options.openInForeground === undefined ? true : option.openInForeground`